patches/minecraft/net/minecraft/block/Block.java.patch

```diff
@@ -1332,6 +1332,50 @@
 +    public void addInformation(ItemStack stack, EntityPlayer player, List<String> tooltip, boolean advanced)
 +    {
 +    }
++    
++    /**
++     * 
++     * @param worldIn The World of the Block
++     * @param pos The position of the Block
++     * @param entityIn The Entity that is requesting the sound
++     * @param type The SoundEvent type that should be returned
++     * @return The sound that will be played
++     */
++    public net.minecraft.util.SoundEvent getSoundEvent(IBlockAccess worldIn, BlockPos pos, @Nullable Entity entityIn, net.minecraftforge.common.EnumSoundEventType type)
```

Comment:
I'd argue that this should return a `SoundType`, which includes many sounds, instead of using a new `EnumSoundEventType` enum to determine what to get.